lib/hdprivatekey.js

```diff
@@ -127,6 +127,11 @@ HDPrivateKey._getDerivationIndexes = function(path) {
   return _.any(indexes, isNaN) ? null : indexes;
 };
 
+HDPrivateKey.prototype.derive = function() {
+  throw new Error('Method has been deprecated, please use deriveChild or deriveNonCompliantChild' +
+                  'and see documentation for more information');
```

Comment:
This is to make sure that if someone upgrades to 1.0.0, that they will not mistakenly be unaware of the bug that affected the deprecated function.